src/main/java/uk/gov/hmcts/reform/blobrouter/data/EnvelopeRepository.java

```diff
@@ -125,4 +125,28 @@ public int markAsDeleted(UUID id) {
             new MapSqlParameterSource("id", id)
         );
     }
+
+    public Integer getEnvelopesCountInCftContainers(Instant fromDateTime, Instant toDateTime) {
```

Comment:
can we not have a parameter for container name and then one method can be used to retrieve both type of envelopes ?

Reply:
but Cft containers are more than 1 container

Reply:
Param can be crime or cft ?

Reply:
I added containers parameter which is a collection of containers. For crime, it is just `crime`, and for cft  - it will be `ServiceConfiguration.getSourceConatiners() minus crime`. 
Is this better?